### PR TITLE
 1️⃣📈 Prevent an off by 1 error for lines without points

### PR DESCRIPTION
**Pull request by 3on**
I made the story a bit more fun.

## With the bug 🇦🇹
image

## Without the bug 🏳
image

### CODE REVIEW

**Comment by janpaul123** [comment]
(We will soon add screenshot testing for this repo so this story is great to prevent future regressions of this bug! 🎉)

File: packages/regl-worldview/src/commands/Lines.js [hunk 1]
```diff
@@ -389,7 +389,8 @@ const lines = (regl: any) => {
 
     const joined = primitive === "line strip";
     const effectiveNumPoints = numInputPoints + (shouldClose ? 1 : 0);
-    const instances = joined ? effectiveNumPoints - 1 : Math.floor(effectiveNumPoints / 2);
+    const guardedNumber = effectiveNumPoints > 0 ? effectiveNumPoints - 1 : 0;
```
Comment (vidaaudrey) [inline packages/regl-worldview/src/commands/Lines.js]: maybe call it `effectiveJoinedInstances` to be more explicit?

File: packages/regl-worldview/src/stories/lines.stories.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+          position: { x: 0, y: 0, z: 0 },
+          orientation: { x: 0, y: 0, z: 0, w: 1 },
+        },
+        points: [],
+        color: [1, 1, 1, 0.75],
+        scale: { x: 2.2, y: 0, z: 0 },
+      },
+    ];
+    return (
+      <Worldview defaultCameraState={DEFAULT_CAMERA}>
```
Comment (vidaaudrey) [inline packages/regl-worldview/src/stories/lines.stories.js]: nit: this story can work without `defaultCameraState={DEFAULT_CAMERA}`

File: packages/regl-worldview/src/stories/lines.stories.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  targetOffset: [5, 5, 0],
+  thetaOffset: Math.PI / 2,
+  phi: Math.PI / 4,
+  distance: 20,
+  targetOrientation: [0, 0, 0, 1],
+};
+
+storiesOf("Worldview", module)
+  .addDecorator(withKnobs)
+  .add("<Lines> Empty points bug", () => {
```
Comment (vidaaudrey) [inline packages/regl-worldview/src/stories/lines.stories.js]: nit: `<Lines> does not render empty points `

File: packages/regl-worldview/src/commands/Lines.js (hunk 1 above)
Comment (jtbandes) [reply packages/regl-worldview/src/commands/Lines.js]: Good catch! Can you simply use (after line 360)
```js
if (numInputPoints > 0) {
  return;
}
```
?

Comment (3on) [reply packages/regl-worldview/src/commands/Lines.js]: Yeap it makes sense to early return.

**Pull request merged**